### ISSUE TITLE
Cancel a student's submission

### ISSUE THREAD

**Issue by lukasc-ubc** [mitodl/edx-sga#139]
As an Instructor, there is no way to cancel a student's submission.  Namely, I'd like to have the ability of allowing a student to re-submit. 

For example, if I give the class an extension, those who submitted early/on-time are at a disadvantage.

The other case if there is an error with the student's file.  It would be nice to have a one-of method of deleting a submission.

This is possible for all other evaluations, via the "Staff Debug Info" button and one of: Reset Student Attempts | Delete Student State.

**Comment by pdpinch** [comment]
Students should be able to change their submission up until the due date, or after the instructor grades the assignment (whichever comes first).

If the due date has passed, and you want the student to to still make a submission, you can change the due date, or set a individual due date extension (IDDE). IDDE is a configuration option in open edX but isn't currently enabled on edge.edx.org or edx.org. 

If the instructor has graded the problem and you want the student to resubmit, I think the best option is to delete the student state. You can do this through the instructor dashboard, student admin tab. The only challenging thing here will be determining the "Problem location" for the SGA XBlock, since it doesn't offer a staff debug button (a shortcoming in the XBlock SDK). 

Let me know if this makes sense, or if I'm misunderstanding what you're asking for.

**Comment by lukasc-ubc** [comment]
Thank you. 

That looks very complicated (resetting the grade), and unfortunately IDDE is not available for me.

Why not add the submission reset as a button or make it part of the behaviour that  you see in the studio editor for that activity?  There already is a means to query student responses and reset them right there, for other types of questions.

**Comment by elisavittoria** [comment]
That's very weird not - not being able to remove an uploaded file. Could you please update about the status of this?

**Comment by pdpinch** [comment]
Sorry, I'm not familiar with the function you mention. It's my understanding that student data isn't available in studio (the course authoring environment).

If there is some documentation you can refer me to, or even just a sample URL, we could investigate further.

**Comment by lukasc-ubc** [comment]
You’re right, it isn’t in studio, but rather in the live version (e.g., edge.edx.org).
The button I refer to is “Staff Debug Info”.

**Comment by lukasc-ubc** [comment]
You are correct.  It appears that this feature was previously unavaialble.  Perhaps it was updated in the live version of edge.edx.org after my inquiry.  I now see an option "Upload a different file”.  This is exactly what I was hoping for.  Thank you!
Lukas
